Sources/OutlineView/OutlineView.swift

```diff
@@ -1,13 +1,29 @@
 import SwiftUI
 import Cocoa
 
+enum ChildSource<Data: Sequence> {
```

Comment:
@ZofiaBernadetta Very true. It is a big change. So should I just add extra description to the pull request at the top of this page?

Reply:
Yes! That would be perfect!

It would finally be nice to include it in the commit message as well. Before merging we can either squash all commits into one, or if you prefer we can have a few commits that each add an individual chunk of the feature and compile/test successfully.

Reply:
Sounds good! I updated the PR message. Let me know if that looks better. As squashing the commits vs breaking them apart... I have no preference. It sounds like squashing would be easier, though, right?

Reply:
The description looks much better!

>It sounds like squashing would be easier, though, right?

Let's go for squashing the commits then. Before merging, you can choose to squash your commits and add a fitting commit message, or if you prefer, I can do that for you.

Reply:
Sounds good. So that can be the last thing after everything else is done, right?

Reply:
Yes!